src/components/Tables/filters/FilterByUserButton.tsx

```diff
@@ -0,0 +1,75 @@
+import { Button } from '@material-ui/core';
```

Comment:
I think this component probably isn't necessary if you go the checkbox route. But if not, I would make this component a more generic one which handles rendering a `BooleanFilterState`